### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andrew-codechimp**
Change HACS installation details as you won't be able to just search HACS until you eventually adds to HACS default repo's (takes months once you submit).

Fix manual link that had a reference to battery notes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,15 +18,18 @@ This is an attempt to pull data and control the local web interface of the Harvs
 ### HACS
 [![Open your Home Assistant instance and open a repository inside the Home Assistant Community Store.](https://my.home-assistant.io/badges/hacs_repository.svg)](https://my.home-assistant.io/redirect/hacs_repository/?owner=yay-adrian&repository=HA-Harvst-Watermate&category=Integration)
 
-Or
-Search for `Harvst Watermate` in HACS and install it under the "Integrations" category.
+Restart Home Assistant
+
+In the HA UI go to "Configuration" -> "Integrations" click "+" and search for "Harvst Watermate"
```
Comment (yayadrian) [inline README.md:23]: It doesn't currently have UI integration so this wouldn't work.

File: README.md [hunk 2]
```diff
@@ -18,15 +18,18 @@ This is an attempt to pull data and control the local web interface of the Harvs
...
 ### Manual Installation
 <details>
 <summary>More Details</summary>
 
-* You should take the latest [published release]([https://github.com/andrew-codechimp/ha-battery-notes/releases](https://github.com/yayadrian/HA-Harvst-Watermate/releases/).  
-* To install, place the contents of `custom_components` into the `<config directory>/custom_components` folder of your Home Assistant installation.  
+* You should take the latest [published release](https://github.com/yayadrian/HA-Harvst-Watermate/releases/).  
+* To install, place the contents of `custom_components` into the `<config directory>/custom_components` folder of your Home Assistant installation.
+* Restart Home Assistant
+* In the HA UI go to "Configuration" -> "Integrations" click "+" and search for "Harvst Watermate"
```
Comment (yayadrian) [inline README.md:32]: It doesn't currently have UI integration so this wouldn't work.

**Review by yayadrian — CHANGES_REQUESTED** [review]
Thanks for the PR. Two issues with the UI integration. It only currently works with configuration.yaml

**Comment by yayadrian** [comment]
@andrew-codechimp thanks for pointing out the incorrect link to your repo. I've now updated the readme to remove this.

**Comment by andrew-codechimp** [comment]
Ah yes, I've never done an integration without a UI config flow.  
The search in HACS won't work though as it's not a HACS default repo.

**Pull request closed without merging**